spec/features/courses/age_range_in_years/edit_spec.rb

```diff
@@ -92,14 +92,8 @@
       )
 
       choose("course_age_range_in_years_other")
-      click_on "Save"
 
-      expect(age_range_in_years_page).to be_displayed
-      expect(age_range_in_years_page.error_flash).to have_content(
-        "You’ll need to correct some information.\nEnter an age for both from and to Enter an age Enter an age",
-      )
-
-      fill_in("course_course_age_range_in_years_other_from", with: "16")
+      fill_in("course_course_age_range_in_years_other_from", with: "14")
```

Comment:
Should we use SitePrism page objects for these? It'd save on effort if/when we change the selector for these ... and kind of feels odd that we use the SitePrism stuff for some things and not others.